.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
Do we need to tie down the distribution?  We don't do it in the master branch.
Or are you trying to use sudo enabled Ubuntu Trusty for some reason?